### PR TITLE
Liberty role migration+alternative download

### PR DESCRIPTION
**Pull request by romakarol**
- tested download path
- ~~did not test copy path yet~~ tested copy path
- a small unrelated change to the iim README

### CODE REVIEW

File: roles/ihs/README.md [hunk 1]
```diff
@@ -24,7 +24,7 @@ NOTE: ihs_admin_pass should be changed after first installation.
 | `ihs_fp_installer_path` | `WAS/9.0.5Fixpacks`                                 |
 | `ihs_fp_installer_archive_list` | `9.0.5-WS-[IHSPLG|WCT]-FP008.zip` |
 | `ihs_pid`               | `v90_9.0.5008.20210525_1601`                        |
-| `ihs_java_zip_path`     | `Java/IBM/ibm-java-sdk-8.0-6.26-linux-x64-installmgr.zip` |
+| `liberty_installer_path`     | `Java/IBM/ibm-java-sdk-8.0-6.26-linux-x64-installmgr.zip` |
```
[the remark this answers is not in the document]
Comment (salob) [reply roles/ihs/README.md:27]: also this looks to be a java installer not a liberty one...

File: roles/liberty/meta/main.yml [hunk 2]
```diff
@@ -0,0 +1,48 @@
+---
+galaxy_info:
+  author: Andrey Zhereshchin
```
Comment (salob) [inline roles/liberty/meta/main.yml:3]: change this to SPM DevOps Team or whatever we have in our other roles

File: roles/liberty/defaults/main.yml [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+
+liberty_version: 21.0.0.9
+
+# Default JVM Options
+liberty_default_heapsize: 1024m
+liberty_enable_verbose_gc: false
+liberty_extra_jvm_options: []
+
+# Common
+profiled_path: /opt/profile.d
```
Comment (salob) [inline roles/liberty/defaults/main.yml:14]: can you please add the commented out download_url and download_header that use in your tasks

File: roles/liberty/meta/main.yml [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+  # https://galaxy.ansible.com/api/v1/platforms/
+  #
+  platforms:
+    - name: EL
+      versions:
+        - 7
+    - name: MacOSX
+      versions:
+        - 10.15
+    - name: Ubuntu
```
Comment (salob) [inline roles/liberty/meta/main.yml:28]: remove

File: roles/liberty/files/AppServer.properties [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+## IT IS VERY IMPORTANT TO USE '/' FOR DIRECTORY PATHS. ##
+
+# Property to indicate whether WLS/SAP or WAS is installed.
+as.vendor=WLP
+
+# The username and password for admin server.
+security.username=websphere
+# Encrypt the plain-text password using 'build encrypt -Dpassword=<password>'
+# Below is the encryption for the default password ("websphere")
+security.password=XOVRjjVTebM8gV953LGMLQ==
```
[the remark this answers is not in the document]
Comment (romakarol) [reply roles/liberty/files/AppServer.properties:12]: this file is in a different location to, but almost exactly the same as websphere/templates/appserverProperties.j2 which I believe we agreed is fine.

[the remark this answers is not in the document]
Comment (romakarol) [reply roles/liberty/files/AppServer.properties:12]: image

**Pull request merged**